src/CoreGraphics/CGColorSpace.cs

```diff
@@ -506,6 +513,25 @@ public NSData GetIccData ()
 				return CGColorSpaceSupportsOutput (handle);
 			}
 		}
+
+		[iOS(10,0)][Mac(10,12)][TV(10,0)][Watch(5,0)]
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern IntPtr CGColorSpaceCopyPropertyList (IntPtr space);
+
+		[iOS(10,0)][Mac(10,12)][TV(10,0)][Watch(5,0)]
+		[DllImport (Constants.CoreGraphicsLibrary)]
+		static extern IntPtr CGColorSpaceCreateWithPropertyList (IntPtr plist);
+
+		[iOS(10,0)][Mac(10,12)]
+		[TV(10,0)][Watch(5,0)]
+		public CFPropertyList ToPropertyList ()
+		{
+			var x = CGColorSpaceCopyPropertyList (handle);
+			if (x == IntPtr.Zero)
+				return null;
+			return new CFPropertyList (x);
```

Comment:
calling that `.ctor` will call retain on something coming out of `*Copy*` (already retained)